fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java

```diff
@@ -1102,51 +1104,86 @@ public PlanFragment visitPhysicalSchemaScan(OptExpression optExpression, ExecPla
-                            }
+                    BinaryPredicateOperator binaryPredicateOperator = (BinaryPredicateOperator) predicate;
+                    if (binaryPredicateOperator.getBinaryType() == BinaryPredicateOperator.BinaryType.EQ) {
+                        switch (columnRefOperator.getName()) {
+                            case "TABLE_SCHEMA":
+                            case "DATABASE_NAME":
+                                scanNode.setSchemaDb(constantOperator.getVarchar());
+                                break;
+                            case "TABLE_NAME":
+                                scanNode.setSchemaTable(constantOperator.getVarchar());
+                                break;
+                            case "BE_ID":
+                                scanNode.setBeId(constantOperator.getBigint());
+                                break;
+                            case "TABLE_ID":
+                                scanNode.setTableId(constantOperator.getBigint());
+                                break;
+                            case "PARTITION_ID":
+                                scanNode.setPartitionId(constantOperator.getBigint());
+                                break;
+                            case "TABLET_ID":
+                                scanNode.setTabletId(constantOperator.getBigint());
+                                break;
+                            case "TXN_ID":
+                                scanNode.setTxnId(constantOperator.getBigint());
+                                break;
+                            case "LABEL":
+                                scanNode.setLabel(constantOperator.getVarchar());
+                                break;
+                            case "JOB_ID":
+                                scanNode.setJobId(constantOperator.getBigint());
+                                break;
+                            case "TYPE":
+                                scanNode.setType(constantOperator.getVarchar());
+                                break;
+                            case "STATE":
+                                scanNode.setState(constantOperator.getVarchar());
+                                break;
+                            case "LOG":
+                                // support full match, `log = 'xxxx'`
+                                // TODO: to be fully accurate, need to escape parameter
+                                scanNode.setLogPattern("^" + constantOperator.getVarchar() + "$");
+                                break;
+                            case "LEVEL":
+                                scanNode.setLogLevel(constantOperator.getVarchar());
+                                break;
+                            default:
+                                break;
+                        }
+                    }
+                    // support be_logs.timestamp filter
+                    if (columnRefOperator.getName().equals("TIMESTAMP")) {
```

Comment:
Question: If I wrote an always false condition like `WHERE TIMESTAMP = x AND TIMESTAMP = y` where the x != y, will the query run correctly? Another case is `WHERE TIMESTAMP > x AND TIMESTAMP >y` where x != y, will the query run correctly?

Reply:
> 

And how about disjunctive conditions: `WEHERE TIMESTAMP=x OR TIMESTAMP=y`

Reply:
At this moment, all predicates will be run twice, one in scanner(pushed down), and one in the execution engine, so for the first case `TIMESTAMP = x AND TIMESTAMP = y where the x != y`, the final result will be correct,   but only one predicate of the same type is pushed down to scanner. This can handle most cases, but maybe not all, it's most practical right now.